server/organizations_test.go

```diff
@@ -138,20 +139,22 @@ func TestService_Organizations(t *testing.T) {
 					AllF: func(ctx context.Context) ([]chronograf.Organization, error) {
 						return []chronograf.Organization{
 							chronograf.Organization{
-								ID:   1337,
-								Name: "The Good Place",
+								ID:     1337,
+								Name:   "The Good Place",
+								Public: true,
```

Comment:
this really shouldn't be `true` since we are planning to only allow the default org to be public, currently

Reply:
:+1: